opm/autodiff/BlackoilTransportModel.hpp

```diff
@@ -158,6 +158,13 @@ namespace Opm {
 
 
 
+        std::vector<V>
+        computeFluidInPlace(const ReservoirState& x,
+                            const WellState& xw,
+                            const std::vector<int>& fipnum)
+        {
+            return asImpl().computeFluidInPlace(x, xw, fipnum);
+        }
```

Comment:
I'll see if I can debug this, but some changes in the last month or two caused flow_sequential to cease working for Norne, so I need to fix that first unfortunately.